### ISSUE TITLE
Given a valid JSON string that ends in a newline, the schema() method emits a warning

### ISSUE THREAD

**Issue by davidcantrell-bb** [mojolicious/json-validator#232]
* JSON::Validator version: 4.10
* Perl version: 5.30.1
* Operating system: Mac OS Catalina

### Steps to reproduce the behavior

```perl
JSON::Validator->new()->schema(q!{ "type": "object" }
!)
```

note the newline before the closing quote.

### Expected behavior
There should be no warning

### Actual behavior
A warning is emitted: `Unsuccessful stat on filename containing newline at /blahblah/JSON/Validator.pm line 317.`

### Suggested fix
Tightly scoped `no warnings 'newline';`, something like ...

```perl
my $loadable = do {
  no warnings 'newline';
  (blessed $source && ($source->can('scheme') || $source->isa('Mojo::File')))
  || -f $source
  || (!ref $source && $source =~ /^\w/);
};

**Comment by abraxxa** [comment]
I just chased this error message for hours until I looked for open issues, thanks!

**Comment by DrHyde** [comment]
There, have a trivial fix :-)

**Comment by jhthorsen** [comment]
This is now fixed in #233.
